signer.js

```diff
@@ -12,41 +12,22 @@ class SignatureSigner {
   }
 
   /**
-   * This method turns a hex value into a Uint8Array
-   * @param {number} hex
+   * This method is used for signing the content
+   * @param {Uint8Array} content
    * @return {Uint8Array}
    */
-  hex2uint8array(hex) {
-    return new Uint8Array(hex.match(/.{1,2}/g)
-        .map((byte) => parseInt(byte, 16)));
-  }
+  sign(content) {
+    const b64utohex = jsrsasign.b64utohex;
```

Comment:
I'm wondering why it is copied to the local variable.
Can it just use `jsrsasign.b64utohex` directly ?

Reply:
No. It is not necessary.
They are just functions hooked in the jsrsasign object.
I'll change them.